lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -135,6 +137,34 @@ def has_instructor_tab(user, course):
         )
         assert has_instructor_tab(org_researcher, self.course)
 
+    @ddt.data(
+        ('staff', False),
+        ('staff', True),
+    )
+    @ddt.unpack
+    def test_discussion_tab(self, access_role, waffle_status):
+        """
+        Verify that the Discussion tab only shows up when Pages & Resources flag is off for course
+        """
+        with override_waffle_flag(ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND, waffle_status):
+            discussion_section = '<li class="nav-item"><button type="button" class="btn-link discussions_management" ' \
+                                 'data-section="discussions_management">Discussions</button></li>'
+
+            user = UserFactory.create(is_staff=access_role == 'global_staff')
+            CourseAccessRoleFactory(
+                course_id=self.course.id,
+                user=user,
+                role=access_role,
+                org=self.course.id.org
+            )
+            set_course_cohorted(self.course.id, True)
+            self.client.login(username=user.username, password="test")
+            response = self.client.get(self.url)
+            if waffle_status:
```

Comment:
Like again, I am not a big fan of having `if/else` conditions in tests - I would suggest that you move the `is_tab_available` flag in the ddt.